deployment/Makefile

```diff
@@ -8,7 +8,45 @@ __check_defined = \
 		$(error Undefined $1))
 
 all:
-	echo "#define SECRET 1234" > global_secrets.h
+	echo "#define APPRIVATE "MIICdgIBADANBgkqhkiG9w0BAQEFAASCAmAwggJcAgEAAoGBAKs0iqKUDhJTpdl9
+Y2jl9m+QkPxpWPt794PP5PdGXFfsnDElxXivx+bVWz1n0ZHmxAT2ObdjNVlgHFlo
+5haycr+s+3oyGlAbHExpptYEEaxWuh8U66A9wAdtgYHRFwCuKEMT4VVzMQXyQ886
+SeFY09BVXUbgOFLhas5AfJ5oEcBXAgMBAAECgYBbJ7iA46+yVlNBv59EZNyXfjOU
+EUtNay3YQsK4XgAg2Fw5VtETdxRKdWMFCNfuhR3tPmRwniPzmFa/zDhl8VFil4mN
+dgr5NsfAA2UCVXRvrpjuxXlfEXT7uGVfPqm13PAHMp2yE8xi8LJDIlIjQn3rWS1T
+rp5iqVe67CFfVQn0kQJBAPK0B5fxPjGlkwtYmtT2T/SQeLqIxysSq67o5Nl1CF/8
+ogyOfrt5EaAoqzc0NuIZm0XUz2x5gEEqFuP5GWWNo/0CQQC0lbxLAtKojtc4NPse
+RXMLsKzscLH+HrLgnWYxVaqbZTCztdFZ3dlujFRa93UMpcTZHt+SfB/hogLTYKzU
+UOPjAkB+cv+10THnKe33rl1U8VPtBsxv201xipO6WIrLJWeMoxJG2oL65/Qf7rBt
+3Z1upYq5kkvTp2YEDYrJkgICYA59AkBsKbusWjZ/S39RZTp8OTDdcadBv4Z9JToT
+G3PYXO1hQkzXdzG72AuiEV99VPbA6pFrUviqrroR1Y5HuYILcZ/tAkEAi1S5Km3A
+VUs7eJok7EaIzm/3pjl4WpsOiy8sF4oZmPe//IzcmchT90GDB01Obdp+A17wQWw7
+er0f4QPHImkKyA=="" > ap_secrets.h
+	echo "#define COMPUBLIC "MIGeMA0GCSqGSIb3DQEBAQUAA4GMADCBiAKBgFk97282KRM8z9TeLbEou24YnXZS
+U1F3iSQyO2xDIFh1VE87Y6tvvC6knMPK2lKCGKMiymQ9nsSNX89i7ZEc5TmX8HGz
+PVJzPNKLOT+hgPDoetPIMvTNpUrcdN13hJbvfqIdF3awFH3lR+dWYXPIoCdx9lxX
+Mx9tzB+7X2ZYql7bAgMBAAE="" > ap_secrets.h
+
+	echo "#define COMPRIVATE "MIICdQIBADANBgkqhkiG9w0BAQEFAASCAl8wggJbAgEAAoGAWT3vbzYpEzzP1N4t
+sSi7bhiddlJTUXeJJDI7bEMgWHVUTztjq2+8LqScw8raUoIYoyLKZD2exI1fz2Lt
+kRzlOZfwcbM9UnM80os5P6GA8Oh608gy9M2lStx03XeElu9+oh0XdrAUfeVH51Zh
+c8igJ3H2XFczH23MH7tfZliqXtsCAwEAAQKBgEuaZdrglmG+1g9vfmQkrszxu4x+
+6CkPTLbGe4MXRzLJi3hoRuL5EQKu4EQa+WSG5JpoIfBUJ8o6Pt3x4HNceHCXQs49
+fdmqTbx7dkPVemJcV80TyMuKOo7HH/11f5dmOLZ/ZVijURl5GBABNZ2upzu19ByW
+NfpedmXMKRc2nIAZAkEAmNg2hk+T7MLFITUpTSDgybP/yi/P9sQYS5oEJ6NqvBuw
+QC/TjSWhVTW34xHDwudTwjMnsmPlLwAqejDU6Q/n7wJBAJV4v4ZEuRHnJ97QpaL1
+Dy5M++WHdxCbd6Na0PNFIwHs2j7b3GZ/T9MJbJuUab8y+x9Z2EXVDGIyBxGFFt3a
+69UCQQCYbL7rcL7XlWFb/ZUjqs8ht9U2gDoIIIUOGBleolTWJVK+oDN8BnP+8yRg
+WZ39aA6ak8NZ3s928NZrpvkdN9SXAkAXH84pnJoxLdzaONdXRahqllhmIzYFaLfy
+mss2x5xp4NXG4cbQSKP110BpNWXCwcVeDU1LSHGZA3paeIeYHRVdAkAe+9X1H7XF
+R5DKA5EcaUeX7iIV9O9V12e+O94QCjdrlbHBCqqZ6WHZtogF3yT3cuc6A6DsGkkN
+b754Bj1lCJ2y"" > com_secrets.h
+	echo "#define APPUBLIC "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCrNIqilA4SU6XZfWNo5fZvkJD8
+aVj7e/eDz+T3RlxX7JwxJcV4r8fm1Vs9Z9GR5sQE9jm3YzVZYBxZaOYWsnK/rPt6
+MhpQGxxMaabWBBGsVrofFOugPcAHbYGB0RcArihDE+FVczEF8kPPOknhWNPQVV1G
+4DhS4WrOQHyeaBHAVwIDAQAB"" > com_secrets.h
+	
```

Comment:
This is great Trinity. I tested it though, and it didn't seem to run. I made some changes and replacing those echos with this seems to get the job done:

```
echo '#define APPRIVATE "MIICdgIBADANBgkqhkiG9w0BAQEFAASCAmAwggJcAgEAAoGBAKs0iqKUDhJTpdl9Y2jl9m+QkPxpWPt794PP5PdGXFfsnDElxXivx+bVWz1n0ZHmxAT2ObdjNVlgHFlo5haycr+s+3oyGlAbHExpptYEEaxWuh8U66A9wAdtgYHRFwCuKEMT4VVzMQXyQ886SeFY09BVXUbgOFLhas5AfJ5oEcBXAgMBAAECgYBbJ7iA46+yVlNBv59EZNyXfjOUEUtNay3YQsK4XgAg2Fw5VtETdxRKdWMFCNfuhR3tPmRwniPzmFa/zDhl8VFil4mNdgr5NsfAA2UCVXRvrpjuxXlfEXT7uGVfPqm13PAHMp2yE8xi8LJDIlIjQn3rWS1Trp5iqVe67CFfVQn0kQJBAPK0B5fxPjGlkwtYmtT2T/SQeLqIxysSq67o5Nl1CF/8ogyOfrt5EaAoqzc0NuIZm0XUz2x5gEEqFuP5GWWNo/0CQQC0lbxLAtKojtc4NPseRXMLsKzscLH+HrLgnWYxVaqbZTCztdFZ3dlujFRa93UMpcTZHt+SfB/hogLTYKzUUOPjAkB+cv+10THnKe33rl1U8VPtBsxv201xipO6WIrLJWeMoxJG2oL65/Qf7rBt3Z1upYq5kkvTp2YEDYrJkgICYA59AkBsKbusWjZ/S39RZTp8OTDdcadBv4Z9JToTG3PYXO1hQkzXdzG72AuiEV99VPbA6pFrUviqrroR1Y5HuYILcZ/tAkEAi1S5Km3AVUs7eJok7EaIzm/3pjl4WpsOiy8sF4oZmPe//IzcmchT90GDB01Obdp+A17wQWw7er0f4QPHImkKyA=="' > ap_secrets.h
echo '#define COMPUBLIC "MIGeMA0GCSqGSIb3DQEBAQUAA4GMADCBiAKBgFk97282KRM8z9TeLbEou24YnXZSU1F3iSQyO2xDIFh1VE87Y6tvvC6knMPK2lKCGKMiymQ9nsSNX89i7ZEc5TmX8HGzPVJzPNKLOT+hgPDoetPIMvTNpUrcdN13hJbvfqIdF3awFH3lR+dWYXPIoCdx9lxXMx9tzB+7X2ZYql7bAgMBAAE="' >> ap_secrets.h
echo '#define COMPRIVATE "MIICdQIBADANBgkqhkiG9w0BAQEFAASCAl8wggJbAgEAAoGAWT3vbzYpEzzP1N4tsSi7bhiddlJTUXeJJDI7bEMgWHVUTztjq2+8LqScw8raUoIYoyLKZD2exI1fz2LtkRzlOZfwcbM9UnM80os5P6GA8Oh608gy9M2lStx03XeElu9+oh0XdrAUfeVH51Zhc8igJ3H2XFczH23MH7tfZliqXtsCAwEAAQKBgEuaZdrglmG+1g9vfmQkrszxu4x+6CkPTLbGe4MXRzLJi3hoRuL5EQKu4EQa+WSG5JpoIfBUJ8o6Pt3x4HNceHCXQs49fdmqTbx7dkPVemJcV80TyMuKOo7HH/11f5dmOLZ/ZVijURl5GBABNZ2upzu19ByWNfpedmXMKRc2nIAZAkEAmNg2hk+T7MLFITUpTSDgybP/yi/P9sQYS5oEJ6NqvBuwQC/TjSWhVTW34xHDwudTwjMnsmPlLwAqejDU6Q/n7wJBAJV4v4ZEuRHnJ97QpaL1Dy5M++WHdxCbd6Na0PNFIwHs2j7b3GZ/T9MJbJuUab8y+x9Z2EXVDGIyBxGFFt3a69UCQQCYbL7rcL7XlWFb/ZUjqs8ht9U2gDoIIIUOGBleolTWJVK+oDN8BnP+8yRgWZ39aA6ak8NZ3s928NZrpvkdN9SXAkAXH84pnJoxLdzaONdXRahqllhmIzYFaLfymss2x5xp4NXG4cbQSKP110BpNWXCwcVeDU1LSHGZA3paeIeYHRVdAkAe+9X1H7XFR5DKA5EcaUeX7iIV9O9V12e+O94QCjdrlbHBCqqZ6WHZtogF3yT3cuc6A6DsGkkNb754Bj1lCJ2y"' > com_secrets.h
echo '#define APPUBLIC "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCrNIqilA4SU6XZfWNo5fZvkJD8aVj7e/eDz+T3RlxX7JwxJcV4r8fm1Vs9Z9GR5sQE9jm3YzVZYBxZaOYWsnK/rPt6MhpQGxxMaabWBBGsVrofFOugPcAHbYGB0RcArihDE+FVczEF8kPPOknhWNPQVV1G4DhS4WrOQHyeaBHAVwIDAQAB"' >> com_secrets.h
```